.github/actions/main.ts

```diff
@@ -4,15 +4,20 @@ import { findLastGoodBuild, WorkflowQueries } from './detection'
 const octokit = new Octokit()
 // const octokit = new Octokit({ auth: process.env.GITHUB_TOKEN });
 const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/')
+const workflow_file_name = 'build.yml'
```

Comment:
Don’t think this passes eslint naming conventions (if they applied to this file) but meh.